### PR TITLE
Use slog instead of logrus for windows_desktop_service

### PR DESCRIPTION
**Pull request by zmb3**

### CODE REVIEW

File: lib/srv/desktop/windows_server.go [hunk 1]
```diff
@@ -306,7 +303,7 @@ func NewWindowsService(cfg WindowsServiceConfig) (*WindowsService, error) {
 	// authenticate with LDAP when the LDAP server name is not correct
 	// in the certificate).
 	if cfg.LDAPConfig.CA != nil && cfg.LDAPConfig.InsecureSkipVerify {
-		cfg.Log.Warn("LDAP configuration specifies both a CA certificate and insecure_skip_verify." +
+		cfg.Logger.WarnContext(context.Background(), "LDAP configuration specifies both a CA certificate and insecure_skip_verify."+
```
Comment (ibeckermayer) [inline lib/srv/desktop/windows_server.go:306]:
```suggestion
		cfg.Logger.Warn("LDAP configuration specifies both a CA certificate and insecure_skip_verify."+
```

It looks like you can use the non-`Context` version of these wherever you're doing a `cfg.Logger.<Level>Context(context.Background(), ...`

image

**Review by ibeckermayer — APPROVED** [review]
Just see my comment re: `context.Background()`

File: lib/srv/desktop/windows_server.go (hunk 1 above)
Comment (zmb3) [reply lib/srv/desktop/windows_server.go:306]: We have a lint rule that prevents the use of the non-context versions. The context provides the ability to log extra request-scoped values.

**Review by probakowski — APPROVED** [review]

**Comment by zmb3** [comment]
No v15 backport for now because I used `cmp.Or` from Go 1.22 here.

**Pull request merged**